chroma-manager/tests/framework/integration/shared_storage_configuration/full_cluster/cluster_setup

```diff
@@ -65,12 +65,13 @@ if ! expect ../install.exp $CHROMA_USER $CHROMA_EMAIL $CHROMA_PASS ${CHROMA_NTP_
 fi
 
 # override /usr/share/chroma-manager/storage_server.repo
-if [ -f /tmp/storage_server.repo.in ]; then
-    # make sure we use the correct lustre though!
-    sed -e \"s/@LUSTRE_SERVER_URL@/${LUSTRE_SERVER_URL//\//\\\\/}/\" \
-        -e \"s/@LUSTRE_CLIENT_URL@/${LUSTRE_CLIENT_URL//\//\\\\/}/\" \
-        < /tmp/storage_server.repo.in > /usr/share/chroma-manager/storage_server.repo
-fi
+# if [ -f /tmp/storage_server.repo.in ]; then
+#     # make sure we use the correct lustre though!
+#     sed -e \"s/@LUSTRE_SERVER_URL@/${LUSTRE_SERVER_URL//\//\\\\/}/\" \
+#         -e \"s/@LUSTRE_CLIENT_URL@/${LUSTRE_CLIENT_URL//\//\\\\/}/\" \
+#         -e 's/manager-for-lustre/manager-for-lustre-devel/g' \
+#         < /tmp/storage_server.repo.in > /usr/share/chroma-manager/storage_server.repo
+# fi
```

Comment:
I guess we need something smarter here so that we can continue to use this optimisation.  Probably should generate/substitute the MFL repo to use from [chroma-manager/tests/framework/utils/defaults.sh](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/467/commits/6c0cf3af708c44f8f3d6eb4e02f938f651d54e44#diff-06a3ab104091298eb2e45220cb3e6da4R113)